docker/daemon/daemon_dest.go

```diff
@@ -95,7 +95,7 @@ func (d *daemonImageDestination) Close() {
 	if !d.committed {
 		logrus.Debugf("docker-daemon: Closing tar stream to abort loading")
 		// In principle, goroutineCancel() should abort the HTTP request and stop the process from continuing.
-		// In practice, though, https://github.com/docker/engine-api/blob/master/client/transport/cancellable/cancellable.go
+		// In practice, though, https://github.com/docker/api/docker/types/blob/master/client/transport/cancellable/cancellable.go
```

Comment:
(Really rather than doing all this research, just vaguely mentioning that some implementations may in some cases, as of January 2017 … might be good enough. The primary point of the comment is the motivation for the `CloseWithError`.)